x-pack/platform/plugins/shared/task_manager/server/lib/get_next_run_at.test.ts

```diff
@@ -151,6 +151,37 @@ describe('getNextRunAt', () => {
     jest.clearAllTimers();
   });
 
+  test('should use now even if dtstart defined in rrule with a fixed time when it is given to calculate the next runAt', () => {
```

Comment:
bad merge? this is a duplicate of the above test

Reply:
Thanks for catching this! 🙏